src/operations/replace-followed-threads-link.ts

```diff
@@ -1,9 +1,11 @@
 import * as SITE from "globals-site";
 import * as T from "text";
 
-export default (e: { followedThreadsLink: HTMLElement }) => {
-    const link = e.followedThreadsLink as HTMLAnchorElement;
-    link.textContent = T.general.my_posts;
+export default (e: { followedThreadsLinkText: HTMLElement }) => {
+    const text = e.followedThreadsLinkText as HTMLSpanElement;
```

Comment:
I prefer not to use type assertions unless they're necessary. In this case, we only use the `textContent` property, so we don't need to specialize the type of `text`.

```suggestion
    const text = e.followedThreadsLinkText;
```